paxexpress_cli/versions/core.py

```diff
@@ -15,7 +17,28 @@
 from ..authentication.core import get_auth_header_and_username
 from paxexpress_cli.repositories import core as repositories_core
 from paxexpress_cli.packages import core as packages_core
-from paxexpress_cli.files import core as files_core
+
+
+def select_from_available_versions(
+    subject: str, repo: str, package: str
+) -> Optional[str]:
+    from paxexpress_cli.versions.core import get_packages_available_versions
```

Comment:
Why move the import into the function if we are importing core functionality? I see this pattern usually only with optional dependencies. If it has a reason to be imported here instead of at the top of the file, please add a comment

Reply:
because of circular import.
I will add a comment.